### ISSUE TITLE
Datadog Log Collection Port is not set properly in jenkins logs

### ISSUE THREAD

**Issue by psimms-r7** [jenkinsci/datadog-plugin#344]
**Describe the bug**

Seeing lots of the below errors in the jenkins log, even though this is optional and I have not enabled log collection

```
2023-05-30 16:11:46.763+0000 [id=79160]	SEVERE	o.d.j.p.d.c.DatadogAgentClient#sendLogs: Datadog Log Collection Port is not set properly
```

Label the issue properly.
- Add `severity/` label.
- Add `documentation` label if this issue is related to documentation changes.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment and Versions (please complete the following information):**
A clear and precise description of your setup:
  - version for this project in use.
  - services, libraries, languages and tools list and versions. 

**Additional context**

**Comment by psimms-r7** [comment]
Is there a chance to prioritise this as it completely spams the logs

**Comment by psimms-r7** [comment]
Can this be prioritised please, it makes debugging other problems more difficult

**Comment by jhaumont** [comment]
Hello  @nikita-tkachenko-datadog 
I saw your fix is merged on master since 2 weeks but the build failed and no new tag has been publish with the fix. Any idea where it will be release ?

**Comment by nikita-tkachenko-datadog** [comment]
Hello @jhaumont, thanks for reaching out. We have to do some additional testing before releasing these changes. We hope to roll out a release before the end of the month.

**Comment by nikita-tkachenko-datadog** [comment]
@jhaumont, we have released the plugin version `8.0.0-beta` today. Would you be interested in trying it out to see if it fixes your issue? Beta plugin versions can be installed from the experimental update center, as described [here](https://www.jenkins.io/doc/developer/publishing/releasing-experimental-updates/#using-the-experimental-update-center).

**Comment by jhaumont** [comment]
Hello
Sorry for the late answer. The error has disappeared. thx for the fix ! :)
